src/pymortests/reductors/paaa.py

```diff
@@ -0,0 +1,39 @@
+# This file is part of the pyMOR project (https://www.pymor.org).
+# Copyright pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (https://opensource.org/licenses/BSD-2-Clause)
+
+import numpy as np
+import pytest
+
+from pymor.reductors.aaa import PAAAReductor
+
+pytestmark = pytest.mark.builtin
+
+np.random.seed(0)
```

Comment:
I'm not sure, but maybe this should be inside the function to fix the `sampling_values` and `samples`.